iphone/cli/commands/_build.js

```diff
@@ -4596,6 +4597,11 @@ iOSBuilder.prototype.copyTitaniumiOSFiles = function copyTitaniumiOSFiles() {
 		path.join(this.platformPath, 'iphone', 'Titanium.xcodeproj', 'xcshareddata', 'xcschemes', 'Titanium.xcscheme'),
 		path.join(this.buildDir, this.tiapp.name + '.xcodeproj', 'xcshareddata', 'xcschemes', name + '.xcscheme')
 	);
+	copyAndReplaceFile.call(
+		this,
+		path.join(this.platformPath, 'iphone', 'Titanium.xcodeproj', 'project.xcworkspace', 'contents.xcworkspacedata'),
```

Comment:
I think we're missing this file from the PR? Jenkins is erroring with

```
UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: ENOENT: no such file or directory, stat '/Users/build/Library/Application Support/Titanium/mobilesdk/osx/8.2.1.v20190930143205/iphone/iphone/Titanium.xcodeproj/project.xcworkspace/contents.xcworkspacedata'
```

Reply:
Fixed!